thorntail-runner/src/main/java/org/wildfly/swarm/runner/Runner.java

```diff
@@ -118,13 +118,11 @@ private static URLClassLoader createClassLoader() throws Exception {
         }
 
         URL jarUrl = fatJar.toURI().toURL();
-        return new URLClassLoader(new URL[]{jarUrl}, null);
+        return new URLClassLoader(new URL[]{jarUrl}, ClassLoaders.getPlatformClassLoader());
     }
 
     private static void buildJar(File fatJar) throws IOException, InterruptedException {
-        String classpath = Arrays.stream(((URLClassLoader) Thread.currentThread().getContextClassLoader()).getURLs())
-                .map(URL::getFile)
-                .collect(Collectors.joining(File.pathSeparator));
+        String classpath = System.getProperty("java.class.path");
```

Comment:
I like how you simplified things here :)

Did you have a chance to check if it also works on Windows and Eclipse?

Reply:
I didn't, no :-/

Actually it wasn't a simplification but a necessity, because the system classloader is no longer a `URLClassLoader` :-)